loom/graphics/gfxQuadRenderer.cpp

```diff
@@ -288,15 +299,17 @@ void QuadRenderer::initializeGraphicsResources()
 
     lmLogInfo(gGFXQuadRendererLogGroup, "Initializing Graphics Resources");
 
+    GL_Context* ctx = Graphics::context();

```

Comment:
OK works for me. Using the #define consistently might be smart but is another task for another day.